src/feedback/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import Toast, { toastPositionValues } from "src/feedback/Toast/Toast";
+import Toaster, { useToaster } from "src/feedback/Toast/Toaster";
+import { Button } from "src/index";
+import { AlertDemo } from "src/feedback/Alert/Alert.stories";
+
+const meta: Meta<typeof Toast> = {
+  component: Toast,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Toast>;
+
+const Container = styled.div`
+  height: 100vh;
+`;
+
+export const ToastDemo: Story = {
+  args: {
+    position: "top-right",
+    ...AlertDemo.args,
+  },
+  argTypes: {
+    position: {
+      options: toastPositionValues,
+      mapping: { toastPositionValues },
+      control: { type: "select" },
+      description: "",
+    },
+    ...AlertDemo.argTypes,
+  },
+  render: (args) => (
+    <Container>
+      <Toast {...args}>{args.children}</Toast>
+    </Container>
+  ),
+};
+
+const ButtonsWrapper = styled(Container)`
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+`;
+
+const Buttons = () => {
+  const toaster = useToaster();
+
+  return (
```

Comment:
If we want to be consistent, you should add a Title here ;-) 
`import { Title } from "src/utils/StorybookComponents/Titles";
`
`<Title>Toast</Title>`